packages/UI/src/components/organisms/cards/PositionCard/PositionCard.tsx

```diff
@@ -97,7 +99,9 @@ const PositionCard = ({
           </Box>
           <Box flexDirection="column" gap="8px" width="fit-content">
             <Typography type="h5">time committment</Typography>
-            <Typography type="pSmall">{content.timeCommittment}</Typography>
+            <Typography type="pSmall">
+              {content.commitmentHoursPerWeek}
+            </Typography>
```

Comment:
why changing those? changing those can result in broken components that use the old props, please revert this change